apis/server/container_bridge.go

```diff
@@ -84,6 +84,11 @@ func (s *Server) getContainer(ctx context.Context, rw http.ResponseWriter, req *
 		mounts = append(mounts, *mp)
 	}
 
+	var hostRootPath string
```

Comment:
Is it necessary to explicitly assign a default value? such as `<unknown>` like https://github.com/alibaba/pouch/blob/master/daemon/mgr/system.go#L71